src/main/java/pl/edu/mimuw/queue/AbstractIntQueue.java

```diff
@@ -2,38 +2,48 @@
 
 public abstract class AbstractIntQueue {
 
-  // TODO: you can make changes with this class fields, constructors
-  //  but also add some methods but the specified methods cannot be changed (you
-  //  can change them not to be abstract and provide some implementation for them,
-  //  but they have to have the same names, arguments and returned values)
-
-  /**
-   * Adds element to the queue.
-   *
-   * @throws NullPointerException if the specified element is null
-   */
+  protected int size;
+  protected IntQueueNode head;
+
+  public AbstractIntQueue() {
+    this.size = 0;
```

Comment:
inicjalizujmy wszystkie pola albo żadnych, bo tak jest zagadka troche czemu akurat to jedno wybrałes